build_projects.yml

```diff
@@ -1,8 +1,10 @@
 trigger:
 - master
 - dev/*
+- staging/*
 
 pr:
+- main
```

Comment:
If our main branch is master, why adding main?

Reply:
Since github started the migration from `master` to `main`, this is a safety measure more-or-less.
It is also an alignment with:
https://github.com/analogdevicesinc/no-OS/commit/07f6e2bc73a5f49e027438d5b91b631853da4ffb